generators/upgrade/index.js

```diff
@@ -48,9 +49,9 @@ module.exports = class extends BaseGenerator {
             desc: 'Upgrade to a specific version instead of the latest',
             type: String
         });
-        // This adds support for a `--target-blueprint-version` flag
-        this.option('target-blueprint-version', {
-            desc: 'Upgrade to a specific blueprint version instead of the latest',
+        // This adds support for a `--target-blueprint-versions` flag
+        this.option('target-blueprint-versions', {
+            desc: 'Upgrade to specific blueprint versions instead of the latest',
```

Comment:
Could you provide an example in the description and/or add more detailed info in the description? 
Indeed this parameter changed a bit. I guess it should be used as `--target-blueprint-versions foo@0.0.1,bar@1.0.2`, right ?

Reply:
Fixed